

















































































100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 68124/68124 [02:44<00:00, 415.15it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:00<00:00, 245.04it/s]
  0%|                                                                                                                     | 0/10 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train_tacred.py", line 300, in <module>
    main()
  File "train_tacred.py", line 294, in main
    dict_meta_test_map = train(args, model, train_features, meta_test_filtered_list)
  File "train_tacred.py", line 129, in train
    batch = next(iter(easy_feature))
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 521, in __next__
    data = self._next_data()
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 561, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
KeyError: 2
Traceback (most recent call last):
  File "train_tacred.py", line 300, in <module>
    main()
  File "train_tacred.py", line 294, in main
    dict_meta_test_map = train(args, model, train_features, meta_test_filtered_list)
  File "train_tacred.py", line 129, in train
    batch = next(iter(easy_feature))
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 521, in __next__
    data = self._next_data()
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 561, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/nethome/mrungta8/.local/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
KeyError: 2
****************************** Starting preprocessing of the dataset ******************************
Total number of easy examples = 61294
Easy examples for which meta-test exists = 10
****************************** Preprocessing completed ******************************